src/coreneuron/nrniv/nrniv_decl.h

```diff
@@ -67,4 +67,10 @@ extern int cellorder_nwarp;
 
 // Mechanism pdata index values into _actual_v and _actual_area data need to be updated.
 enum Layout { SoA = 0, AoS = 1 };
+
+// Blocks associated with BEFORE and AFTER have different values based on type.
+// There values are based on neuron/coreneuron implementation details.
+struct BAType {
+    enum { Breakpoint = 1, Solve = 2, Initial = 3, Step = 4, Before = 10, After = 20 };
+};
```

Comment:
@alkino : these are used somewhere? Or added for documentation?

Reply:
Will be use by NMODL in codegen.

Looks at : https://github.com/BlueBrain/nmodl/issues/802